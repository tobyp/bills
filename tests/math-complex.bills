#!./bills
A	B.	(2+3)*5-3/-7
