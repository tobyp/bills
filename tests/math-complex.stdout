todo: B pays 25.428572 to A
